jwql/tests/test_database_interface.py

```diff
@@ -25,7 +25,7 @@
 import string
 
 from jwql.database import database_interface as di
-from jwql.utils.constants import ANOMALIES
+from jwql.utils.constants import ANOMALIES_PER_INST
```

Comment:
@tnking97 Thank you for thinking about the unit tests!  It is something we always tend to forget about.